### ISSUE TITLE
Provide victims with Internet as an option in TUI

### ISSUE THREAD

**Issue by sophron** [wifiphisher/wifiphisher#837]
This mode of Wifiphisher is getting more and more popular. It probably makes sense to re-name the "Scenario Selection" phase to a more generic "Mode of Operation Selection" phase that will include this mode (of providing victims with Internet) too.

**Comment by anakin1028** [comment]
We currently let the users provide `-iI eth0` option to use this mode. To provide the option in TUI we also need to help users select the internet card in TUI right?

**Comment by blackHatMonkey** [comment]
Yeah a question of whether the user wants this option followed by card selection.

**Comment by djdisikd** [comment]
If provide victims with internet we need 3 WiFi adapters?

**Comment by blackHatMonkey** [comment]
@djdisikd Not necessarily. You can use the ethernet interface or specify a second wireless adapter.

**Comment by djdisikd** [comment]
Ok

**Comment by sophron** [comment]
@anakin1028 Yes, correct.
